src/main/resources/messages/messages.properties

```diff
@@ -239,6 +243,11 @@ crowdin.tm.download.source-language-id=Defines source language in the language p
 crowdin.tm.download.target-language-id=Defines target language in the language pair
 crowdin.tm.download.to=Path the translation memory should be downloaded to
 
+# CROWDIN TM DOWNLOAD COMMAND
+crowdin.bundle.download.usage.description=Download bundle
+crowdin.bundle.download.usage.customSynopsis=@|fg(green) crowdin bundle download|@ [CONFIG OPTIONS] [OPTIONS]
+crowdin.bundle.download.id=Bundle identifier
```

Comment:
do we need this? we already have the `crowdin download bundle` described here - https://github.com/crowdin/crowdin-cli/pull/549/files#diff-820c44c8b99ca8885beb6f0cbde7a1897d458a35bedcfe0e79596e64be241b66R58